src/Workspaces/CSharp/Portable/Simplification/CSharpSimplificationService.cs

```diff
@@ -31,7 +28,8 @@ internal partial class CSharpSimplificationService : AbstractSimplificationServi
                 new CSharpParenthesesReducer(),
                 new CSharpEscapingReducer(),
                 new CSharpMiscellaneousReducer(),
-                new CSharpInferredMemberNameReducer());
+                new CSharpInferredMemberNameReducer(),
+                new CSharpDefaultExpressionReducer());
```

Comment:
I'm not clear what this does, since the "complexify" code does not expand `default` to `default(<type>)`. What breaks if this line is removed?